src/ExcalidrawView.ts

```diff
@@ -3287,6 +3290,18 @@ export default class ExcalidrawView extends TextFileView {
                 });
                 if(typeof res === "boolean" && res === false) return false;
               }
+
+              // Disables Middle Mouse Button Paste Functionality on Linux
+              if(
+                !this.modifierKeyDown.ctrlKey 
+                && typeof event !== "undefined"
+                && event !== null 

```

Comment:
Did this since when I was logging the "*event*" object it returned undefined / null when I used the excalidraw right click paste option **or** Ctrl+V.
It only actually returned a ClipboardEvent when I used the middle mouse button click for some reason so I added that as a check